### PR TITLE
graph update

### PR DESCRIPTION
**Pull request by micbar**
## Description
Update Libre Graph Api

## Motivation and Context
Use the latest version of the spec

## How Has This Been Tested?
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by micbar** [comment]
@rhafer @butonic our user spec is different than the  spec from https://github.com/yaegashi/msgraph.go

He has a weird `Object` property.

@rhafer can you check if our spec  https://owncloud.dev/libre-graph-api/ is good, then i could refactor this.

File: go.mod [hunk 1]
```diff
@@ -55,7 +55,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/thejerf/suture/v4 v4.0.1
 	github.com/urfave/cli/v2 v2.3.0
-	github.com/yaegashi/msgraph.go v0.1.4
+	github.com/yaegashi/msgraph.go v0.1.1-0.20200221123608-2d438cf2a7cc
```
Comment (wkloucek) [inline go.mod:58]: is that downgrade intended?

Comment (micbar) [reply go.mod:58]: nope

**Review by refs — APPROVED** [review]

**Comment by refs** [comment]
Agree with @wkloucek , that downgrade has to be reverted.

**Pull request merged**